common/config/presets/windows.toml

```diff
@@ -400,3 +400,7 @@ monerod_password=""
 
 ## Will start mining only when base node has reported that it is bootstrapped
 #mine_on_tip_only=true
+
```

Comment:
The minimal settings for the standalone miner to run should not be commented out.

Reply:
Actually node will start even when no configuration is specified, as it will use defaults.

Reply:
Done